store/sqlstore/upgrade.go

```diff
@@ -762,3 +764,16 @@ func upgradeDatabaseToVersion520(sqlStore SqlStore) {
 		saveSchemaVersion(sqlStore, VERSION_5_20_0)
 	}
 }
+
+func upgradeDatabaseToVersion521(sqlStore SqlStore) {
+	if shouldPerformUpgrade(sqlStore, VERSION_5_20_0, VERSION_5_21_0) {
+		sqlStore.CreateColumnIfNotExists("IncomingWebhooks", "SecretToken", "varchar(26)", "varchar(26)", "")
+		sqlStore.CreateColumnIfNotExists("IncomingWebhooks", "HmacAlgorithm", "varchar(12)", "varchar(12)", "")
```

Comment:
See earlier comments, but I'm wondering if we can support exactly one model to start, and skip all of this modelling complexity (since the model will be implicit).

Reply:
please see the earlier response. Do you suggest to support a signed webhook from only one sender? In this case, we can preset and except for SecretToken, we don't need input from channel admins/webhook owners.